openlibrary/macros/LoanStatus.html

```diff
@@ -1,4 +1,4 @@
-$def with (page, user, editions_page=False, block_name='', render_preview_floater=True)
+$def with (page, user=None, editions_page=False, block_name='', render_preview_floater=True)
```

Comment:
`user` should be fetchable by `web.ctx.user` -- we do **want** the ability to pass it i, but having it default to `None` makes sense.